docs/package.json

```diff
@@ -24,9 +24,9 @@
     "https-localhost": "^4.7.1",
     "unocss": "^0.46.0",
     "unplugin-vue-components": "^0.22.9",
-    "vite": "^3.2.3",
-    "vite-plugin-pwa": "0.13.1",
-    "vitepress": "1.0.0-alpha.27",
+    "vite": "^4.0.0",
+    "vite-plugin-pwa": "0.13.3",
```

Comment:
pwa plugin not yet ready for Vite 4: I've sent rn the PR to the repo, maybe tmr I can release it

Reply:
Docs appear to work fine anyway. We can merge this now, and update plugin later.